pkg/version/version_test.go

```diff
@@ -0,0 +1,45 @@
+package version
+
+import (
+	"testing"
+)
+
+func TestGet(t *testing.T) {
+	tests := []struct {
+		name      string
+		version   string
+		wantMajor string
+		wantMinor string
```

Comment:
could you just have one version object and check that it's equal to the expected? The diff in the output of that check will show whether it's the major or minor that's wrong

Reply:
a lot of the fields need to be set to do a reflectEqual, let me push a separate commit to get your feedback

Reply:
let me know what you think

Reply:
any feedback?